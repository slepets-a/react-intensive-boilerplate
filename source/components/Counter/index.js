import React from "react";
import Styles from "./styles.m.css";

const Counter = ({ count }) => (
    <p className = { Styles.counter }>Post count: {count}</p>
);

export default Counter;
