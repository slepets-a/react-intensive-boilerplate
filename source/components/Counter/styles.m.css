.counter {
    display: flex;
    justify-content: center;
    position: relative;
    margin-bottom: .5rem;
    color: var(--paletteColor7);

    &::after {
        content: '';
        position: absolute;
        top: -6px;
        left: -35px;
        width: 30px;
        height: 30px;
    }
}