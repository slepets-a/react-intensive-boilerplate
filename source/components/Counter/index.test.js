import dom from 'react-test-renderer';

import Counter from './index';

const renderTree = dom.create(<Counter count = { 3 } />).toJSON();

test('Counter should correspond to its snapshot counterpart', () => {
    expect(renderTree).toMatchSnapshot();
});
