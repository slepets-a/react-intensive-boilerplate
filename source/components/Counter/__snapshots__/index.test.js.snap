// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter should correspond to its snapshot counterpart 1`] = `
<p
  className="counter"
>
  Post count: 
  3
</p>
`;
